execution/executor/tests/internal_indexer_test.rs

```diff
@@ -192,6 +193,208 @@ fn test_db_indexer_data() {
             total_version,
         )
         .unwrap();
-    let address_one_kv_res: Vec<_> = address_one_kv_iter.collect();
-    assert_eq!(address_one_kv_res.len(), 152);
+    let address_one_kv_res = address_one_kv_iter.collect::<Result<Vec<_>, _>>().unwrap();
+
+    let (code, resources): (Vec<_>, Vec<_>) = address_one_kv_res
+        .into_iter()
+        .map(|(s, _)| s)
+        .partition(|s| s.is_aptos_code());
+
+    let expected_code = vec![
```

Comment:
Are these changes related? Not familiar with the code here and don't know what it is trying to achieve.

Reply:
there is a check that all resources published in genesis are indexed correctly.

here we added mem.move, so it failed. failing on 153>152 is confusing.

so I just pulled the list of resources - and am asserting on them, and failure will tell which resource is different. 

cc @areshand